source/_docs/tools/quick-bar.markdown

```diff
@@ -71,14 +57,67 @@ Currently-supported commands:
+
+We know something like "**light.ch**" should match "**light.ch**andelier". Similarly, "**telev**" should match "media_player.**telev**ision".
+
+But with Quick Bar, "**lich**" would also match "**li**ght.**ch**andelier", and "**plyrtv**" would also match "media_**pl**a**y**e**r**.**t**ele**v**ision". It checks letter *sequences* rather than exact substrings.
+
+One nice use-case for this is that you can quickly filter out an entire domain of entities with just a couple letters and a period. For example, "**li.**" will match any "**light.***" entities. Continuing with "li.ch" would bring up the chandelier right away.
+
+### Filters work against friendly name too
+
+If "light.hue_ceiling_light" has been named "Chandelier", you can type either "hue_ceil" or "chand" to find it.
+
+### Use the enter key any time to open the top result in the list
+
+As soon as the item you wanted shows up at the top of your filtered results, just hit "enter" to activate it -- no need to arrow down to the item, or click with your mouse.
+
+### Use arrow keys to move around the list
+
+When in the text field, use the down arrow `↓` to navigate down the item list. Hit `enter` to activate the currently-highlighted row.
+
+When in the item list, use the up arrow `↑` to navigate up the item list, and to get back into the text field.
+
+### Typing more letters will always add to your filter string
+
+Say you've just used arrow keys to navigate half-way down the list, and want to add more text to your filter. You don't need to click back into the text field, just start typing new letters and they'll append to your filter.
+
+## Troubleshooting
+
+### Dialog doesn't launch using hotkeys
+
+There are a few possible reasons why the quick bar dialog won't launch:
+
+1. Your user is not an admin.
+2. The application lost focus. Try clicking into the main content area of the application and typing the shortcut again.
+3. You have disabled Keyboard Shortcuts in your User Profile settings.
+4. Shortcut is marked by browser as non-overridable. Firefox does this with some shortcuts, for example. But this shouldn't be a problem with single-key shortcuts currently used by the Quick Bar.
+5. Some other application or browser extension is using or overriding the shortcut. Try disabling the extension.
+
+### A command is missing
+
+The command list only shows commands that are available to you based on your user settings, and loaded integrations.
+
+For example, if you don't have `automations:` in your config, then you won't see the "Reload Automations" command.
+
+If "Advanced Mode" is turned off in User Settings, then any command related to advanced mode will not appear in the list.
 
-In Developer Tools -> States, if you're trying to find "light.chandelier" you would have to type out a complete, exact substring to match it (e.g., "light.cha" or "chandel").
+If a command is missing that you feel is in error, please create an issue on GitHub.
```

Comment:
Sorry, dropping one more thing. We should remove this line. Mainly because it applies to about every page we have (and we should not add it to every page we have).

Furthermore, this can be interpreted as an invitation for making feature requests as issues on GitHub (while we are trying to move to discussions for those).

```suggestion
```

Reply:
Done